FirebaseCore/Extension/FIRLogger.h

```diff
@@ -116,6 +116,12 @@ extern void FIRLogInfo(FIRLoggerService service, NSString *messageCode, NSString
 extern void FIRLogDebug(FIRLoggerService service, NSString *messageCode, NSString *message, ...)
     NS_FORMAT_FUNCTION(3, 4);
 
+// Swift does not support variadic function calls
+extern void FIRLogDebug1(FIRLoggerService service,
```

Comment:
maybe name this `FIRLogDebugSwift`? the `1` is somewhat inscrutable.

Reply:
Done. And found a more general solution. Thanks for the poke.